rust/Cargo.toml

```
@@ -36,6 +36,7 @@ serde_repr = "0.1"
 thiserror = "1.0.30"
 time = "0.3"
 url = "2.2"
+tokio = { version = "1.36.0", features = ["time"] }
```

```suggestion
tokio = { version = "1.36.0", default-features = false, features = ["time"] }
```
Just to avoid bloating clients?

---

Tokio doesn't have any default features.